packages/discord.js/typings/index.d.ts

```diff
@@ -5959,7 +5968,15 @@ export interface GuildMemberEditOptions {
   reason?: string;
 }
 
-export type GuildResolvable = Guild | NonThreadGuildBasedChannel | GuildMember | GuildEmoji | Invite | Role | Snowflake;
+export type GuildResolvable =
+  | Guild
+  | NonThreadGuildBasedChannel
+  | GuildMember
+  | PartialGuildMember
```

Comment:
I don't think it makes much sense to resolve a Guild from a partial member, but that one's up to debate I'd say